h5py/_selector.pyx

```diff
@@ -59,14 +66,16 @@ cdef class Reader:
         cdef:
```

Comment:
The docscring on the line above is now wrong (Selector not Reader) but the UI won't let me suggest changes that far from a diff.

Reply:
Thanks, good catch.